bitbots_vision/docs/manual/yoeo_vision.rst

```diff
@@ -60,13 +59,9 @@ looks like this:
 
 Of course, the actual class names need to be adapted to your specific YOEO variant. The following names are supported:
 
-* Object detection: 'ball', 'goalpost', 'robot', 'robot_red', 'robot_blue', 'robot_unknown'
+* Object detection: 'ball', 'goalpost', 'robot_red', 'robot_blue', 'robot_unknown'
 * Semantic segmentation: 'background', 'lines', 'field'
 
-If 'team_colors' is set to true, the robot class has to be split into 'robot_red', 'robot_blue' and 'robot_unknown'. In
```

Comment:
Isn't this still the case?

Reply:
Good point! I must have forgotten to rollback that file too.